ports/atmel-samd/mpconfigport.mk

```diff
@@ -62,6 +62,12 @@ ifndef CIRCUITPY_SAMD
 CIRCUITPY_SAMD = 1
 endif
 
+ifndef MICROPY_PY_ULAB
+ifneq ($(CIRCUITPY_SMALL_BUILD), 1)
```

Comment:
Our practice has been to omit the space after the `,`. It's allowed but spaces are significant in ways that aren't expected, e.g.
```make
FOO = 12
ifeq ($(FOO), 12 )
$(error true)
else
$(error false)
endif
```
prints `false` because of the space after the `12`
```make
ifeq ($(FOO),12)
# is same as
ifeq ($(FOO), 12)
# is not same as
ifeq ($(FOO),12 )
```

Reply:
Fixed, thanks!